tests/sentry/monitors/test_tasks.py

```diff
@@ -13,182 +13,215 @@
     MonitorType,
     ScheduleType,
 )
-from sentry.monitors.tasks import check_monitors
+from sentry.monitors.tasks import TIMEOUT, check_monitors
 from sentry.testutils import TestCase
 
 
 class CheckMonitorsTest(TestCase):
+    def make_ref_time(self):
```

Comment:
Updated all the tests to use this helper which produces timestamps more accurate to what production would look like.